tests/test_simulation.py

```diff
@@ -2015,32 +2028,32 @@ def test_stress_skin(self, frame_modal_simulation: post.ModalMechanicalSimulatio
         )
         assert len(result.columns.set_ids) == 1
         if SERVERS_VERSION_GREATER_THAN_OR_EQUAL_TO_7_1:
-            assert len(result.index.mesh_index) == 36
```

Comment:
how confident are we all these unit tests changes?

Reply:
did you plot all of these to make sure they were good?

Reply:
@cbellot000 Yes, I debugged most of them locally to check the actual number of elements involved and whether the new reference is correct this time. This is why I switched to NamedSelections for most of them.

Reply:
I think we can now duplicate all these tests once with reduce_mesh=True, where we should have the same results as before, right?

Reply:
@cbellot000 I started making specific tests for when we use `reduce_mesh`, but I do not think we need to duplicate all of the already existing ones for each result.